archive_viewer/mixins/plot_config.py

```diff
@@ -0,0 +1,53 @@
+from widgets import ColorButton
+from pyqtgraph import ViewBox
+
+
+class PlotConfigMixin:
```

Comment:
Style: Can you add docstrings to all the methods in this class? 

for reference on the docstring format we are using:  
https://numpydoc.readthedocs.io/en/latest/format.html